vendor/github.com/bitrise-io/go-xcode/v2/xcconfig/xcconfig.go

```diff
@@ -2,34 +2,57 @@ package xcconfig
 
 import (
 	"fmt"
+	"path/filepath"
+	"strings"
+
 	"github.com/bitrise-io/go-utils/v2/fileutil"
 	"github.com/bitrise-io/go-utils/v2/pathutil"
-	"path/filepath"
 )
 
 // Writer ...
 type Writer interface {
-	Write(content string) (string, error)
+	Write(input string) (string, error)
 }
 
 type writer struct {
 	pathProvider pathutil.PathProvider
 	fileManager  fileutil.FileManager
+	pathChecker  pathutil.PathChecker
 }
 
 // NewWriter ...
-func NewWriter(pathProvider pathutil.PathProvider, fileManager fileutil.FileManager) Writer {
-	return &writer{pathProvider: pathProvider, fileManager: fileManager}
+func NewWriter(pathProvider pathutil.PathProvider, fileManager fileutil.FileManager, pathChecker pathutil.PathChecker) Writer {
+	return &writer{pathProvider: pathProvider, fileManager: fileManager, pathChecker: pathChecker}
 }
 
-func (w writer) Write(content string) (string, error) {
+// Write writes the contents of input into a xcconfig file if
```

Comment:
Write writes

Reply:
Can not fix it here, needs a go-xcode update.